sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -314,16 +314,24 @@ def process_algorithms(
                     l1b_calibration_data,
                     ialirt_calibration_data,
                 )
-            elif instrument == "codicelo":
+            elif instrument == "codice_lo":
                 logger.info("Processing CoDICE-Lo.")
-                download_path = get_ancillary("codice", "l1a-sci-lut")
-                logger.info("codice sci-lut: %s", download_path)
-                result, _ = process_func(combined, download_path)
-            elif instrument == "codicehi":
+                l1a_download_path = get_ancillary("codice", "l1a-sci-lut")
+                l2_download_path = get_ancillary("codice", "l2-hi-ialirt-efficiency")
+                logger.info("codice sci-lut: %s", l1a_download_path)
+                logger.info("codice sci-lut: %s", l2_download_path)
```

Comment:
These shouldn't both be the same thing. Maybe 
```suggestion
                logger.info("codice sci-lut: %s", l1a_download_path)
                logger.info("codice efficiency: %s", l2_download_path)
```